docs/workflow/trimming/feature-switches.md

```diff
@@ -21,8 +21,8 @@ configurations but their defaults might vary as any SDK can set the defaults dif
 | CustomResourceTypesSupport | System.Resources.ResourceManager.AllowCustomResourceTypes | Use of custom resource types is disabled when set to false. ResourceManager code paths that use reflection for custom types can be trimmed. |
 | EnableUnsafeBinaryFormatterInDesigntimeLicenseContextSerialization | System.ComponentModel.TypeConverter.EnableUnsafeBinaryFormatterInDesigntimeLicenseContextSerialization | BinaryFormatter serialization support is trimmed when set to false. |
 | BuiltInComInteropSupport | System.Runtime.InteropServices.BuiltInComInterop.IsSupported | Built-in COM support is trimmed when set to false. |
-| EnableCPlusPlusCLIHostActivation | Internal.Runtime.InteropServices.InMemoryAssemblyLoader.IsSupported | C++/CLI host activation code is disabled when set to false and related functionality can be trimmed. |
-| _EnableCallingManagedFunctionFromNativeHosting | Internal.Runtime.InteropServices.ComponentActivator.IsSupported | Getting a managed function from native hosting is disabled when set to false and related functionality can be trimmed. |
+| EnableCPlusPlusCLIHostActivation | System.Runtime.InteropServices.CPlusPlusCLIHostActivation.IsSupported | C++/CLI host activation code is disabled when set to false and related functionality can be trimmed. |
+| _EnableCallingManagedFunctionFromNativeHosting | System.Runtime.InteropServices.CallingManagedFunctionFromNativeHosting.IsSupported | Getting a managed function from native hosting is disabled when set to false and related functionality can be trimmed. |
```

Comment:
`CallingManagedFunctionFromNativeHosting` is a bit specific. I would instead call it `ConsumingManagedCodeFromNativeHosting`. This is not limited to simple functions and in some cases types may be rooted as well. I would instead focus on the fact the problem is accessing logic or data from a native environment.